packages/lib/gpcircuits/package.json

```diff
@@ -44,6 +44,7 @@
     "@pcd/tsconfig": "0.11.1",
     "@pcd/util": "^0.5.1",
     "@semaphore-protocol/identity": "^3.15.2",
+    "@semaphore-protocol/identity-v4": "npm:@semaphore-protocol/identity@4",
```

Comment:
I'm not familiar with this syntax, but I'm okay with it if it works.  Ideally this PR should be consistent with Richard's PR for Semaphore V4 elsewhere.
The other alternative approach is to use @semaphore-protocol/core which contains identity and other stuff, though I don't know if that would increase bundle size.